admin_pages/general_settings/General_Settings_Admin_Page.core.php

```diff
@@ -243,22 +239,29 @@ public function load_scripts_styles()
      */
     public function admin_init()
     {
-        EE_Registry::$i18n_js_strings['invalid_server_response'] = wp_strip_all_tags(__(
-            'An error occurred! Your request may have been processed, but a valid response from the server was not received. Please refresh the page and try again.',
-            'event_espresso'
-        ));
-        EE_Registry::$i18n_js_strings['error_occurred']          = wp_strip_all_tags(__(
-            'An error occurred! Please refresh the page and try again.',
-            'event_espresso'
-        ));
-        EE_Registry::$i18n_js_strings['confirm_delete_state']    = wp_strip_all_tags(__(
-            'Are you sure you want to delete this State / Province?',
-            'event_espresso'
-        ));
-        $protocol                                                = is_ssl() ? 'https://' : 'http://';
+        $this->core_config = EE_Registry::instance()->CFG->core;
+
+        EE_Registry::$i18n_js_strings['invalid_server_response'] = wp_strip_all_tags(
+            esc_html__(
+                'An error occurred! Your request may have been processed, but a valid response from the server was not received. Please refresh the page and try again.',
+                'event_espresso'
+            )
+        );
+        EE_Registry::$i18n_js_strings['error_occurred']          = wp_strip_all_tags(
+            esc_html__(
+                'An error occurred! Please refresh the page and try again.',
+                'event_espresso'
+            )
+        );
+        EE_Registry::$i18n_js_strings['confirm_delete_state']    = wp_strip_all_tags(
```

Comment:
guessing that's a filterable i18n string that's being passed to JS via an object dumped to the DOM, therefore it needs to have all tags stripped else it will break the JS